sso_cookies.py

```diff
@@ -26,9 +26,7 @@ def transform(cookies_string):
           "Now you can run `safaribooks.py` without the `--cred` argument...")
 
 
-USAGE = "\n\n[*] Please use this command putting as argument the cookies retrieved by your browser.\n" + \
-        "[+] In order to do so, please follow these steps: \n" + \
-        "https://github.com/lorenzodifuccia/safaribooks/issues/150#issuecomment-555423085\n"
+USAGE = {"_mkto_trk":"id:107-FMS-070&token:_mch-oreilly.com-1616464224230-70388","cd_user_id":"1785cc6a491128-0c0392a0739809-5771031-144000-1785cc6a4923c0","_vwo_uuid_v2":"DB24A37707F9A1192ACBA4E0F1BB8A780|b7bc3ebba5c6406c00e16a730bc6cf77","_vis_opt_s":"1%7C","_vis_opt_test_cookie":"1","_vwo_uuid":"D7412610CB35C24E65242D3705F7C859B","_vwo_ds":"3%241636112987%3A61.78316817%3A%3A","_gid":"GA1.2.1950975615.1636112989","BrowserCookie":"b3576ed8-408f-495a-93a2-15ff81f08482","logged_in":"y","salesforce_id":"38d445507d6469045e3e632a5e23214f","csrfsafari":"5zM2kINWOkCnYmK5vamHm8y9IRNmVhuro5fRUyMYLjcP7IWspbz5zjKzyp9MfYjO","sessionid":"zjbz1u9mwp2jpfv6d6lhw1av7t3poxt7","_evga_5802":"{%22uuid%22:%226eb9acf713f4d579%22%2C%22puid%22:%22IX147rEPxw75xw4UehPSqGNHoX87FWtg1kUTAzcCJ2fARGxuhXaVdbifnglILEqS2sKXoatJi_xCEZkGlBnfHtAfYahrMph28n5D5P-6YLSoDglupri1i4ejSyER8yKO%22}","OptanonAlertBoxClosed":"2021-11-05T11:50:46.157Z","_ga_092EL089CH":"GS1.1.1636113504.2.0.1636113507.57","orm-jwt":"eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImI1ZjliMGU1YzM1ZDRiY2NjYjY1YzZkOGQxYzQ2MWI5In0.eyJhY2N0cyI6IFsiMzg0ZDhkZmYtOGQ5Yi00YWU4LWEzMWYtZWNlZTljYTcxMjYzIl0sICJlaWRzIjogeyJleGFjdHRhcmdldCI6ICJwbGF0Zm9ybV9wcm9kXzA3M2YyNjQ3LWEzMjItNDExYy05ZTRiLTFkOTg3ZDNmYjRkYyIsICJoZXJvbiI6ICIwNzNmMjY0Ny1hMzIyLTQxMWMtOWU0Yi0xZDk4N2QzZmI0ZGMifSwgImVudiI6ICJwcm9kdWN0aW9uIiwgImV4cCI6IDE2MzYxMjQ3MjIsICJpbmRpdmlkdWFsIjogZmFsc2UsICJwZXJtcyI6IHsiYWNhZG0iOiAidiIsICJhcGlkYyI6ICJ2IiwgImFzaWduIjogImNldiIsICJjbmZyYyI6ICJ2IiwgImNwcmV4IjogInYiLCAiY3NzdGQiOiAidiIsICJlcHVicyI6ICJ2IiwgImxycHRoIjogInYiLCAibHZ0cmciOiAidiIsICJudGJrcyI6ICJ2IiwgIm9yaW9sIjogInYiLCAicGx5bHMiOiAiY2V2IiwgInNjbnJpbyI6ICJ2IiwgInVzYWdlIjogImMiLCAidXNycGYiOiAiZXYiLCAidmlkZW8iOiAidiJ9LCAic3ViIjogIjA3M2YyNjQ3LWEzMjItNDExYy05ZTRiLTFkOTg3ZDNmYjRkYyJ9.MjlTrx0tdO8dT-4_EXM0BE0UEkMwTMuxajuwWBNuFfo0LrL7qHqcerUMsSn4I_Badk8QDbesES9KtYRqRErjInxg4H9cGe_nFStOenN6dRxwSrtiFHRqg3nIn4yz2ZzRGlBwnNxWKAP4fY29Zy-0OlmcU8C3AoJwEZdyd5Hz08I","amp_49f7a6":"aCsMH9qUWfLcZMGM0YUc-6.MDczZjI2NDctYTMyMi00MTFjLTllNGItMWQ5ODdkM2ZiNGRj..1fjo6m77e.1fjo8gl78.9.e.n","_ga_4WZYL59WMV":"GS1.1.1636121124.2.1.1636123039.59","_ga":"GA1.1.1853110738.1616464224","OptanonConsent":"isGpcEnabled=0&datestamp=Fri+Nov+05+2021+14%3A37%3A19+GMT%2B0000+(Greenwich+Mean+Time)&version=6.25.0&isIABGlobal=false&hosts=&consentId=18e92cf9-4ad5-4e67-a64a-bd9769d27b00&interactionCount=1&landingPath=NotLandingPage&groups=C0001%3A1%2CC0002%3A1%2CC0003%3A1%2CC0004%3A1&AwaitingReconsent=false&geolocation=GB%3BENG","_dd_s":"rum=0&expire=1636124031889&logs=1&id=ee6e38d6-44de-47a1-b46c-9ae3603b1dbc&created=1636121123980"}
```

Comment:
It's not a good idea to post sensitive content on GitHub... I suggest you to remove this commit

Reply:
please help